ts/features/idpay/code/screens/IDPayResultScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import { IOPictograms } from "@pagopa/io-app-design-system";
+import { useNavigation, useRoute } from "@react-navigation/native";
+import * as React from "react";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+
+export const IDPayCodeResultsScreen = () => {
```

Comment:
LGTM! Just a small change and we are good to do the merge :)
Also, file name should be `IdPayCodeResultsScreen.tsx`

```suggestion
export const IdPayCodeResultsScreen = () => {
```

Reply:
as with the other PR's comment, are we sure we want to use that notation? as of right now the saga PR is using the one I'm using here, also so far this is the notation we've used if I'm not mistaken

Reply:
The PR you are citing is using `IdPay` prefix. Also we decided to go with `IdPay` prefix as the new standard: acronyms should be treated as any other word and follow the camelcase rule.

Reply:
the PR I am citing does not at the moment completely use that casing, most likely because since macOS has ignoreCasing by default, so git behaves in the same way by not pushing changes to filename casing [see here](https://github.com/pagopa/io-app/pull/4995/files?diff=unified&w=0#diff-645f71601b11168d39161e4b3d43ddef4706c73337fff56fd57c3e52f4315fe4) | [and here](https://stackoverflow.com/questions/71230581/gits-not-detecting-file-name-casing-change) .
that said, addressed 👍

Reply:
To change the case you need to first rename the file to something like "FileName2.tsx" and then "fileName.tsx". I know, it sucks